src/server/acl/helpers.cc

```diff
@@ -181,6 +181,13 @@ std::optional<User::UpdatePass> MaybeParsePassword(std::string_view command, boo
   return {};
 }
 
+std::optional<std::string> MaybeParseNamespace(std::string_view command) {
+  if (absl::StartsWith(command, "TENANT:")) {
+    return std::string(command.substr(7));
+  }
```

Comment:
what if it's empty namespace? is there a way to indicate an error?
(can also be a TODO)

Reply:
It's fine if the namespace is empty. It's just the default. It's valid to specify `TENANT:` (i.e. empty namespace string)
However this comment makes me think that using the term `TENANT` isn't great, because this feature is better known as namespaces. I'll change that.

Reply:
Maybe use `NS` then? namespace is too verbose, imho

Reply:
I did as you asked, but I actually don't think it's a great idea.
NS could mean different things (name server, nano seconds, Nova Scotia :rofl:), and this string is not supposed to be used a lot anyway

Reply:
I wish Dragonfly could teleport users to Nova Scotia! (LGTM)